runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -3282,6 +3284,9 @@ MM_CopyForwardScheme::completeScan(MM_EnvironmentVLHGC *env)
 			case MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_BREADTH_FIRST:
 				completeScanCache(env);
 				break;
+			case MM_GCExtensions::OMR_GC_SCAVENGER_SCANORDERING_DYNAMIC_BREADTH_FIRST:
```

Comment:
Don't have to worry about Balanced GC, yet. But anyhow, you can colapse the first two cases.